recipes/cvbio/cvbio.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
```

Comment:
This was copied from: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/fgbio/fgbio.py